src/modules/vendor/pages/ni/management/CoursesDirectory.vue

```diff
@@ -144,18 +154,20 @@ export default {
         this.companyOptions = [];
       }
     },
+    updateCourseCompany () {
+      if (this.newCourse.type === INTRA) delete this.newCourse.company;
```

Comment:
if (this.newCourse.type === INTER_B2B) plutot :), ou if (this.newCourse.type !== INTRA), ou if (!isIntraCourse) jsp